composer.json

```diff
@@ -12,12 +12,12 @@
         }
     ],
     "require": {
-        "typo3/cms-backend": "^12.2 || dev-main",
```

Comment:
I would prefere to require "^12.2 || dev-main" as we use this extension on the main branch of the documentation which always uses the latest main branch

Reply:
Nope, when we tackled the last issues we should release a stable version. Which should only be compatible with v12.4, not with the upcoming v13. When we branch then we can add dev-main again (then without 12.4 compatibility):

Reply:
The current main branch is not aiming to develop TYPO3 13, it has bugfixes of TYPO3 v12.4. The Core did not yet branch of v12 to 13

Reply:
Yup, but there will be no new features for 12.4, so 12.4 is sufficient.

Reply:
Bugfixes in the code, even those of comments in the code can and do positively influence the code-blocks we are printing. therefore I would always use dev-main for the main documentation branches

Reply:
Okay, no problem, will close the PR for now.

Reply:
Shouldn't we require "^12.4 || dev-main"? then?

Reply:
Open a new PR. I do not agree. So the closing.